src/Database/Builder.php

```diff
@@ -69,30 +68,29 @@ protected function searchWhereInternal($term, $columns, $mode, $boolean)
         }
 
         if ($mode === 'exact') {
-            $this->where(function ($query) use ($columns, $term) {
+            $this->where(function (Builder $query) use ($columns, $term) {
```

Comment:
Is this referring to itself? Why is it required?

Reply:
Yes it's referring to itself, because a callback is default mixed typed.

Reply:
Right, but what benefit is this specific change adding?

Reply:
Autocompletion for ide's and let the developer know which type is returned

Reply:
@LukeTowers any update on this?

Reply:
LGTM, thanks!